ros2param/ros2param/verb/set.py

```diff
@@ -28,6 +29,18 @@
 from ros2param.verb import VerbExtension
 
 
+class RequireParameterPairAction(argparse.Action):
+    """Argparse action to validate parameter argument pairs."""
+
+    def __call__(self, parser, args, values, option_string=None):
+        if len(values) == 0:
+            parser.error('No parameters specified')
+            SystemExit(2)
```

Comment:
parser.error is able to handle system error 2?

Reply:
@fujitatomoya  You are correct, I forgot to delete that line after I realized parser.error will handle it. I'll fix your review comments & add tests soon